### DISCUSSION TITLE
MTEX Workshop 2024

### DISCUSSION THREAD

**Discussion by ralfHielscher** [mtex-toolbox/mtex#1948] [Announcements]
Dear MTEX community,

we are extremely happy to announce the MTEX workshop 2024 in Freiberg. With the new MTEX version 6.0 in the pipeline we have many exciting improvements to present and are also very eager to learn from your applications of MTEX. As usual the first week will be an introductory course to texture analysis and MTEX. Following the feedback from previous year we will directly start with EBSD analysis and provide the theoretical background as needed.

More information can be found at the workshop page https://mtex-toolbox.github.io/workshop24 which will be constantly updated.

Participation will be possible in person and online. However, everybody who attended the workshop in person will tell you - it was worth the additional effort :smile: 

All the best,
The MTEX team.

**Comment by SunilAnandatheertha** [comment]
When does the registration open ?

**Reply by ralfHielscher** [reply]
Registration is now open!

**Reply by dansilvasouza** [reply]
Dear @ralfHielscher , I have registered but I didn't recieve any confirmation or invoice yet. I need an invoice for reibursement purposes. Do you have any information about how long it takes to get an online confirmation of the registration? Thank you!

**Comment by sarialzerey** [comment]
Is there a way to pay the fees other than bank transaction? like online link or so? I need the receipt as well. Please help with this.

**Comment by lebraunjames** [comment]
Dear @ralfHielscher, 

I hope everything is going well on your side. we recently sent you an email about sending 5 PhD students for this MTEX Workshop. We were hoping to pay the bill all at once via credit card, and if you can issue a receipt/invoice at once, that would be great. Please send us a reply when you find the time.

**Reply by ralfHielscher** [reply]
Hi,
unfortunately, I could not find you email. Could you please resend it to me? You students are very welcome at the workshop. All the best,

Ralf.

**Reply by lebraunjames** [reply]
It's all good, as we received a reply yesterday and have now paid. Thanks!

**Comment by bamobe** [comment]
Dear @ralfHielscher,

I hope this message finds you well.
I recently tried accessing the materials for the previous workshops via the provided link "https://www-user.tu-chemnitz.de/~rahi/mtexWorkshop17ICOTOM/index.php", but unfortunately, it seems to be broken.

Could you kindly provide an alternative link or method for obtaining these files?

Thank you very much for your attention to this matter.

**Reply by ralfHielscher** [reply]
I'm sorry for that. I lost access to this site as well. I try what I can do.
Ralf.
